phpBB/assets/javascript/installer.js

```diff
@@ -310,9 +380,28 @@
 	}
 
 	/**
+	 * Refresh page
+	 */
+	function doRefresh() {
+		resetPolling();
+
+		// Debug
+		console.log ('doRefresh(): ' + $(location).attr('pathname'));
```

Comment:
This will break older browsers, please ~~get a build tool to remove it~~ wrap it in a check to make sure it exists